packages/template-retail-react-app/app/constants.js

```diff
@@ -139,3 +139,24 @@ export const REMOVE_UNAVAILABLE_CART_ITEM_DIALOG_CONFIG = {
     }),
     onPrimaryAction: noop
 }
+
+export const SUPPORTED_STORE_LOCATOR_COUNTRIES = [
+    {
+        countryCode: 'US',
+        countryName: defineMessage({
+            defaultMessage: 'United States',
+            id: 'store_locator.dropdown.united_states'
+        })
+    },
+    {
+        countryCode: 'DE',
+        countryName: defineMessage({
+            defaultMessage: 'Germany',
+            id: 'store_locator.dropdown.germany'
+        })
+    }
+]
```

Comment:
If a project wanted to add or change the list of supported stores, what would they need to do? Would they modify this const?

I am wondering if it might be better to have this in a stores.js config file (similar to how we have a sites.js for defining the sites)?

Reply:
Actually, wait. The list of supported stores is probably defined in BM for each site, no? So a project would be setting these there?

Reply:
Yes the stores are specified by the customer in BM and are retrieved in our app via a hook of the API endpoint [searchStores](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-stores?meta=searchStores)

Reply:
Thanks @jeremy-jung1. So this const and the ones below it are temporary and will be replaced with a hook to call searchStores in a subsequent PR?

If so, could you leave a comment saying that the constants are temporary?

Reply:
@vcua-mobify Ah no this const on the other hand is the list of countries that the customer can configure the store locator to query. It is meant to be a permanent configuration option. On top of that, the configurations below the list of countries specify default parameters for the initial query to be done server side. 

I had a discussion with @vmarta and @bendvc about whether or not these configurations should go in the sites.js, and the `constants.js` file was deemed most appropriate due to it being the current pattern of putting new configurations and also in anticipation for a future refactor of the template-retail-react-app, in which it would be easier if at many configurations as possible came from the same file.